features/support/worlds/benefit_application_world.rb

```diff
@@ -108,7 +108,7 @@ def terminate_application(application, date)
   def reinstate_application(application)
     terminate_application(application, application.end_on - 4.months)
     application.reload
-    EnrollRegistry[:benefit_application_reinstate].feature.stub(:is_enabled).and_return(true)
+    allow(EnrollRegistry[:benefit_application_reinstate].feature).to receive(:is_enabled).and_return(true)
     EnrollRegistry[:benefit_application_reinstate].setting(:offset_months).stub(:item).and_return(12)
```

Comment:
honest question, this one doesnt throw the deprecation error ?

Reply:
that's a valid point, it will throw the deprecation warning. Although there are still N number of references left to convert, I am planning to target those in next PR. This itself is a giant one.